src/ui/components/RecommendedBadge/styles.scss

```diff
@@ -0,0 +1,57 @@
+@import '~ui/css/styles';
+
+$recommended-text-color: $orange-60;
+$recommended-hover-color: $orange-50;
```

Comment:
Can we go for a darker color? I don't really see the difference..

Reply:
I suppose so. I doubt these will be the final colors. I am still waiting to hear back from @brassy- about which colors to use. FTR, I found the difference pretty obvious when hovering over the lint, but I'll change it.